core/js/typespecs/app_types.ts

```diff
@@ -138,6 +138,7 @@ export const aiSecretSchema = z
   .object({
     id: aiSecretBaseSchema.shape.id,
     created: aiSecretBaseSchema.shape.created,
+    updated_at: datetimeStringSchema.nullish(),
```

Comment:
Maybe we should add an `updated_at` field to `generateBaseTableSchema`, so that we can use it elsewhere and include docs for the field.